src/game/enums.ts

```diff
@@ -9,3 +9,7 @@ export enum Spell {
   Abara = "abara",
   Kadabara = "kadabara"
 }
+
+export enum CheatCode {
```

Comment:
Maybe we need to add the ninja turtles 2 cheat code too

Reply:
Definitely not - I'll never remember it.

Reply:
But not with unblocked, you'll never forget because it will be in the codebase forever!:

`B A B A Up Down B A Left Right B A Start`

Reply:
try sending some messages here

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/3ba26e47-f387-4c68-a5ef-43303d4503e7?message=575a7450-269f-4ed5-8130-352056976df4).

Reply:
Ok, 

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/3ba26e47-f387-4c68-a5ef-43303d4503e7?message=70d4afaa-25db-402e-85e4-a969c53e0a58).

Reply:
ok

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/3ba26e47-f387-4c68-a5ef-43303d4503e7?message=f655a508-b461-4102-890c-18ea37cd0e13).

Reply:
And now from VS Code

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/3ba26e47-f387-4c68-a5ef-43303d4503e7?message=363c8c94-5b1f-43dd-9fc0-2ddf330619f1).

Reply:
That's right! Forgot about that one...

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/50095a61-63bc-4b9f-b0a9-14dacb71228b/thread/caf0f27e-5afd-4eca-9dbc-d474756298fd?message=e1a02b0a-0b09-4219-83cb-5644ba641a3e).

Reply:
Please test notifications!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/50095a61-63bc-4b9f-b0a9-14dacb71228b/thread/caf0f27e-5afd-4eca-9dbc-d474756298fd?message=a59f21e1-3df1-4c56-9fea-779ef5f3ea1c).

Reply:
Hi Jon!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/50095a61-63bc-4b9f-b0a9-14dacb71228b/thread/caf0f27e-5afd-4eca-9dbc-d474756298fd?message=d551452f-326b-4f31-a6c5-02b82bf425b1).

Reply:
I think we should create a cheat code that is a little more difficult to discover!